api-reference/arcana/streaming-wav.mdx

```diff
@@ -87,19 +89,15 @@ headers = {
     "Content-Type": "application/json"
 }
 
-# Use stream=True to handle streaming response
-response = requests.request("POST", url, json=payload, headers=headers, stream=True)
+with requests.post(url, headers=headers, json=payload, stream=True) as response:
+    response.raise_for_status()
+    audio_bytes = b''
+    for chunk in response.iter_content(chunk_size=4096):
+        if chunk:
+            audio_bytes += chunk
 
-# Check if the request was successful
-if response.status_code == 200:
-    with open("audio_output.wav", "wb") as f:
-        for chunk in response.iter_content(chunk_size=1024):
-            if chunk:
-                f.write(chunk)
-else:
-    print("Error:", response.status_code)
-
-print(response.text)
+audio_samples = np.frombuffer(audio_bytes, dtype=np.int16)
```

Comment:
@MaCaki also want to get your opinion on one thing, Arcana wav streaming is a bit different from Mist, it streams back a WAV header and then PCM chunks, see: https://github.com/rimelabs/rime-deploy-baseten-arcana/blob/f6c80dd6770d455880f29a6f591f7783bf6c45e1/truss/entrypoint/stream_audio.py#L95

That means if we directly saves it to a wav file:
```
with open("audio_output.wav", "wb") as f:
    f.write(audio_bytes)
```
This will error out. 

We have two options:
1. keep this change and show customers how to save a proper WAV using numpy and soundfile libraries, so this sample will be different from PCM, MP3 and MULAW.
2. Remove the saving part completely, just show how to get audio in bytes.